packages/desktop-client/src/components/budget/BudgetCategories.jsx

```diff
@@ -250,6 +253,7 @@ export const BudgetCategories = memo(
               content = (
                 <ExpenseCategory
                   cat={item.value}
+                  categoryGroup={categoryGroup}
```

Comment:
Maybe we can set a `group` property here: https://github.com/actualbudget/actual/pull/2582/files#diff-53ee16f0a6c14399478cb1429e6dbe32375b4289b1fba1acff9ead903adc2c36R68 and just set this to `categoryGroup={item.group}`?